webapp/src/hocs/authRedirects.js

```diff
@@ -20,6 +20,11 @@ export const redirectIfAuthed = redirectUnless(
 );
 
 export const redirectIfUnauthed = redirectUnless(
-  "/sign-in",
+  "/start",
   ({ userAuthed }) => userAuthed
 );
+
+export const redirectIfUnboarded = redirectUnless(
+  "/onboarding",
+  ({ user }) => user.onboarded
+);
```

Comment:
@rgalanakis Let me know if there's any modifications (if any) that would make sense here.

Reply:
This is a great idea.